app/models/solidus_acima/gateway.rb

```diff
@@ -13,7 +21,33 @@ def authorize(_amount, payment_source, _options)
       )
     end
 
-    def capture(*args); end
+    def capture(_amount, checkout_token, options)
```

Comment:
It looks to me that the second parameter is called `response_code` everywhere [solidus](https://github.com/solidusio/solidus/blob/e878076f2ed670d07654ab6293a16588743f2fa6/core/app/models/spree/payment/processing.rb#L59), [solidus_square](https://github.com/nebulab/solidus_square/blob/main/app/models/solidus_square/gateway.rb#L21) [solidus_paypal_braintree](https://github.com/solidusio/solidus_paypal_braintree/blob/master/app/models/solidus_paypal_braintree/gateway.rb#L143)